File: src/main/java/compiler/exceptions/IndentationException.java
package compiler.exceptions;

public class IndentationException extends RuntimeException{

    public IndentationException(){ super(); }
    public IndentationException(String message) { super(message); }
    public IndentationException(String message, Throwable cause) { super(message, cause); }
    public IndentationException(Throwable cause) { super(cause); }

}

File: src/main/scala/test_classes/exceptions/IndentationException.scala
package test_classes.exceptions

class IndentationException() extends RuntimeException {
  def this(message: String) {
    this()
    println(message)
  }

  def this(message: String, cause: Throwable) {
    this()
    println(message)
  }

  def this(cause: Throwable) {
    this()
    cause.getStackTraceString
  }
}